### PR TITLE
Cleaning in pautomorphicform.py

### PR DESCRIPTION
**Pull request by Eloitor**
Remove unused variable in pautomorphicform.py

### :memo: Checklist

- [ x] The title is concise, informative, and self-explanatory.
- [x ] The description explains in detail what this PR is about.

### CODE REVIEW

**Review by S17A05 — APPROVED** [review]
LGTM.

**Comment by Eloitor** [comment]
I added 2 new commits removing another variable and an unused parameter that was also undocumented, so I think it is save to remove.

File: src/sage/modular/btquotients/pautomorphicform.py [hunk 1]
```diff
@@ -1912,18 +1910,15 @@ def integrate(self, f, center=1, level=0, method='moments'):
         R2 = PolynomialRing(f.base_ring(), 'x')
         x = R2.gen()
         value = 0
-        ii = 0
         if method == 'riemann_sum':
             for e in E:
-                ii += 1
                 exp = ((R1([e[1, 1], e[1, 0]])) ** (self.parent()._U.weight()) * e.determinant() ** (-(self.parent()._U.weight()) / 2)) * f(R1([e[0, 1], e[0, 0]]) / R1([e[1, 1], e[1, 0]]))
                 # exp = R2([tmp[jj] for jj in range(self.parent()._k-1)])
                 new = eval_dist_at_powseries(self.evaluate(e), exp.truncate(self.parent()._U.weight() + 1))
```
Comment (S17A05) [inline src/sage/modular/btquotients/pautomorphicform.py]: Does the comment in the middle line contain any valuable information or may it be removed?

**Review by S17A05 — COMMENTED** [review]
If you add commits after a positive review, please (have someone) reset the review label in the future or put those changes into a new PR instead. 
The changes look good to me, but while you're at it I noticed a few more things:
- Can't the index `ii` in `.coleman` also be removed?
- There are some overly long lines in the file - could you please break them up into multiple lines?

File: src/sage/modular/btquotients/pautomorphicform.py [hunk 2]
```diff
@@ -2124,21 +2117,21 @@ def coleman(self, t1, t2, E=None, method='moments', mult=False,
...
-            sage: D = -11 # optional - magma
-            sage: X.is_admissible(D) # optional - magma
+            sage: X = BruhatTitsQuotient(p,lev)
+            sage: k = 2
+            sage: M = X.harmonic_cocycles(k,prec)
+            sage: B = M.basis()
+            sage: f = 3*B[0]
+            sage: MM = X.padic_automorphic_forms(k,prec,overconvergent = True)
+            sage: D = -11
+            sage: X.is_admissible(D)
```
Comment (mkoeppe) [inline src/sage/modular/btquotients/pautomorphicform.py]:
```suggestion
            sage: X = BruhatTitsQuotient(p, lev)
            sage: k = 2
            sage: M = X.harmonic_cocycles(k, prec)
            sage: B = M.basis()
            sage: f = 3*B[0]
            sage: MM = X.padic_automorphic_forms(k, prec, overconvergent=True)
            sage: D = -11
            sage: X.is_admissible(D)
```

File: src/sage/modular/btquotients/pautomorphicform.py [hunk 3]
```diff
@@ -2124,21 +2117,21 @@ def coleman(self, t1, t2, E=None, method='moments', mult=False,
...
-            sage: P = Kp.gen() # optional - magma
-            sage: Q = 2+Kp.gen()+ p*(Kp.gen() +1) # optional - magma
-            sage: F = MM.lift(f) # long time, optional - magma
-            sage: J0 = F.coleman(P,Q,mult = True) # long time, optional - magma
+            sage: K.<a> = QuadraticField(D)
+            sage: Kp.<g> = Qq(p**2,prec)
+            sage: P = Kp.gen()
+            sage: Q = 2+Kp.gen()+ p*(Kp.gen() +1)
+            sage: F = MM.lift(f) # long time
+            sage: J0 = F.coleman(P,Q,mult = True) # long time
```
Comment (mkoeppe) [inline src/sage/modular/btquotients/pautomorphicform.py]:
```suggestion
            sage: K.<a> = QuadraticField(D)
            sage: Kp.<g> = Qq(p**2, prec)
            sage: P = Kp.gen()
            sage: Q = 2 + Kp.gen() + p*(Kp.gen()+1)
            sage: F = MM.lift(f)  # long time
            sage: J0 = F.coleman(P, Q, mult=True)  # long time
```

**Review by mkoeppe — APPROVED** [review]
LGTM

**Pull request merged**